src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,152 @@
+package duke.task;
+
+import duke.exception.DukeException;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    /** Start date and time of event */
+    private LocalDateTime startDateTime;
+    /** End date and time of event */
+    private LocalDateTime endDateTime;
+
+    /**
+     * Constructs Event class.
+     *
+     * @param description Description of task.
+     * @param start Start date and time of task.
+     * @param end End date and time of task.
+     * @throws DukeException If dates entered are invalid.
+     */
+    public Event(String description, String start, String end) throws DukeException {
+        super(description);
+        LocalDate startDate = null;
+        LocalDateTime startDateTime = null;
+        LocalDate endDate = null;
+        LocalDateTime endDateTime = null;
+        DateTimeFormatter[] formatters = {
+                DateTimeFormatter.ofPattern("ddMMyyyy HHmm"),
```

Comment:
Good job on including so many datetime formats for Event class even when it is not one of the required increments 👍